packages/vuetify/src/util/helpers.ts

```diff
@@ -468,3 +467,21 @@ export function findChildren (vnode?: VNodeChild): ComponentInternalInstance[] {
 
   return []
 }
+
+export function pick<
```

Comment:
Duplicate of `extract` on 185

Reply:
We talked about `pick` having better typing.